lib/sles4sap/sap_deployment_automation_framework/deployment.pm

```diff
@@ -142,7 +142,7 @@ sub az_login {
     assert_script_run("source $temp_file");
 
     my $login_cmd = 'while ! az login --service-principal -u ${ARM_CLIENT_ID} -p ${ARM_CLIENT_SECRET} -t ${ARM_TENANT_ID}; do sleep 10; done';
-    assert_script_run($login_cmd, timeout => 5 * 60);
+    assert_script_run($login_cmd, timeout => 30);
```

Comment:
5*60 for login is just too much. 
Needed to change this for simulating login failure, so I don't have to 5min.